web/src/components/dropdowns/match-dropdown.tsx

```diff
@@ -0,0 +1,87 @@
+import { Dropdown, DropdownItem, MenuToggle, MenuToggleElement } from '@patternfly/react-core';
+import { LongArrowAltDownIcon, LongArrowAltUpIcon, RouteIcon } from '@patternfly/react-icons';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Match } from '../../model/flow-query';
+
+export interface MatchDropdownProps {
+  selected: Match;
+  setMatch: (l: Match) => void;
+  id?: string;
+}
+
+export const MatchValues = ['any', 'all', 'peers'] as Match[];
+
+export const MatchDropdown: React.FC<MatchDropdownProps> = ({ selected, setMatch, id }) => {
+  const { t } = useTranslation('plugin__netobserv-plugin');
+  const [isOpen, setOpen] = React.useState(false);
+
+  const getMatchDisplay = (layoutName: Match) => {
+    switch (layoutName) {
+      case 'any':
+        return t('Any');
+      case 'all':
+        return t('All');
+      case 'peers':
+        return t('Peers');
+      default:
+        return t('Invalid');
+    }
+  };
+
+  const getIcon = (layoutName: Match) => {
+    switch (layoutName) {
+      case 'any':
+        return (
+          <>
+            <LongArrowAltUpIcon />
+            <LongArrowAltDownIcon />
+          </>
+        );
+      case 'all':
+        return <LongArrowAltUpIcon />;
+      case 'peers':
+        return <RouteIcon />;
```

Comment:
- I'd put "all" on top, it should be the default (at least it has always been)
- The Up/Down icons look more appropriate for peers (which we may rename as "bidirectional")
- But I can't see which icon would really work to distinguish any/all , then... ; if we don't find something, we can still remove those icons and keep only text.

Reply:
(I mean, putting All on top is for L13 not here)